notebooks/book1/03/gauss_infer_2d.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def make_graph2(x1,x2,p,savename,title,fig=None,ax=None):
</pre><p>Function can be renamed as make_contour_graph() ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Okay I will change according to the above mentioned comments, but I was confused about the x,y labels for the subplots.</p>

Reply:
You can update label as following
|fig | xlabel | ylabel |
|-- | -- | -- |
| a) | y1 | y1 |
| b) | z1 | z2 |
| c) | z1 | z2 |

let me know what do you think?